common/djangoapps/entitlements/migrations/0002_auto_20171031_1553.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('entitlements', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AlterField(
```

Comment:
Alter operations in Migrations sometimes have dangerous behavior because of our blue/green deployment. I'm not sure if this case would cause a problem.

Reply:
I can ask the DevOps team.  There is no data in the Table as it is not in use yet, but I will check in to make sure.